<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>723</width>
    <height>491</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>723</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>地铁线路维护</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>151</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>请选择线路：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="lineComboBox">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="addLine">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>添加线路</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icon/icon/line.png</normaloff>:/icon/icon/line.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="stationTable">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>515</width>
         <height>380</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>380</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>515</width>
         <height>380</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>408</y>
         <width>511</width>
         <height>37</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="manageLine">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>管理线路信息</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/icon/settings.png</normaloff>:/icon/icon/settings.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addStation">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>添加站点</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/icon/pos.png</normaloff>:/icon/icon/pos.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>公告</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="announcement">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>发布公告</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icon/icon/announcement.png</normaloff>:/icon/icon/announcement.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
